### PR TITLE
Added a really chill dino :tada:

### PR DESCRIPTION
**Pull request by markonije**
Added a really chill dino :tada:

### CODE REVIEW

**Review by exu3 — CHANGES_REQUESTED** [review]
😎 cool! Could you add this dinosaur to the README please?

**Comment by markonije** [comment]
Added it!
@exu3

File: README.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ _The Dinosaur Who Can Never Look Back_
 
 ![](/purple_dino.png)
 
+"A very chill dino"
+
+![](/markonije_chill.png)
```
Comment (exu3) [inline README.md:37]:
```suggestion
![](markonije_chill.png)
```

**Review by exu3 — APPROVED** [review]
very chill, thank you!

**Pull request merged**